Resources/Prototypes/Entities/Structures/Piping/Atmospherics/binary.yml

```
@@ -375,6 +375,7 @@
   - type: PipeColorVisuals
   - type: Rotatable
   - type: GasRecycler
+  #- type: PipeRestrictOverlap # Allow pipe stacking for MRP
```

YAMLlint (1.35.1)

[warning] missing starting space in comment

(comments)